nav-app/src/app/datatable/data-table.component.html

```diff
@@ -496,6 +496,113 @@
+                                            placeholder="label"
+                                            [(ngModel)]="linkEditField[i].label"
+                                            (input)="updateLinks()">
+                                    </textarea>
+                                </mat-form-field>
+                                <mat-form-field>
+                                    <textarea matInput
+                                            matTextareaAutosize
+                                            placeholder="url"
+                                            [(ngModel)]="linkEditField[i].url"
+                                            (input)="updateLinks()">
+                                    </textarea>
+                                </mat-form-field>
+                                <button class="button" (click)="removeLink(i)">remove</button>
+                            </div>
+                            
+                            <div class="divider" (click)="onClickDivider(i, !link.divider, 'links')" *ngIf="canAddDivider(i, 'links') || link.divider"
+                                matTooltipPosition="below" [matTooltip]="link.divider ? 'remove divider' : 'add a divider'">
+                                <div [ngClass]="{'opt-divider': canAddDivider(i, 'links'), 'set-divider': link.divider}">
+                                    <mat-divider></mat-divider>
+                                    <button mat-icon-button class="icon">{{link.divider ? '-': '+'}}</button>
+                                    <mat-divider></mat-divider>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="add-more">
+                        <button class="button" (click)="addLink()">add {{linkEditField.length ? 'another' : 'a'}} link</button>
+                    </div>
+                </div>
+             </div>
+
+
+            <!-- technique metadata -->
+            <div *ngIf="configService.getFeature('metadata')"  class="control-row-item">
+
+                <div class="control-row-button dropdown noselect"
+                     (click)="viewModel.isCurrentlyEditing() && viewModel.metadataMatch ? currentDropdown = currentDropdown !== 'metadata' ? 'metadata' : null : continue"
+                     matTooltipPosition="below"
+                     [matTooltip]="viewModel.metadataMatch ? 'metadata' : 'cannot edit metadata on selected techniques with differing metadata'"
+                     [matTooltipDisabled]="!viewModel.isCurrentlyEditing()"
+                     [style.color]="viewModel.isCurrentlyEditing() && viewModel.metadataMatch ? '#000000' : '#aaaaaa'">
+                     <img [src]="viewModel.isCurrentlyEditing() && viewModel.metadataMatch ? 'assets/icons/ic_metadata_black_24px.svg' : 'assets/icons/ic_metadata_gray_24px.svg'" alt="metadata" />
```

Comment:
To work with dark theme this image needs to use material icons instead of importing an image from assets directly.